### ISSUE TITLE
Panic in formatter

### ISSUE THREAD

**Issue by manio** [0x20F/paris#44]
Hi,
Sorry, that is't me again bothering you... I hope you don't mind :)

I think I found a panic bug (I found it by accident when trying to use paris logger in a place of env_logger in cec project) :)

The following line:
```rust
log.info("<< powering on 'TV' (0)");
```

is causing:
```
thread 'main' panicked at 'byte index 23 is out of bounds of `< powering on 'TV' (0)`', /usr/src/paris/src/formatter/keys/key_list.rs:43:35
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by 0x20F** [comment]
Hi!
No worries, it's good that someone finds these bugs :D 

I'll try looking into this during the weekend and see how I can solve it.
Thank you for the issue!

**Comment by 0x20F** [comment]
Will be solved in `1.5.10` which will be released later today.
Sorry it took so long :D

**Comment by manio** [comment]
Not a problem - i just temporarily used `env_logger` for my project.
But of course I'll be glad to return to the proper one :D
Thanks :)
